src/Applications/LDAS_App/ldas_setup

```diff
@@ -1405,7 +1428,11 @@ def _printExeInputKeys(rqdExeInpKeys):
     print '#  Surface meteorological forcing time step is in seconds. #'
     print '#                                                          #'     
     print '############################################################'
-
+    print '# for adas-coupling exp (COUPLING = 1) '
+    print '# MET_TAG:     ${ADAS_exp}__Nx+- '
+    print '# the following link needs to be created'
+    print '# after ldas exp setup: in ../input/met_forcing/'
```

Comment:
move these help comments up to where COUPLING is defined

Reply:
moved to where LADAS_COUPLING is defined